Lesson5/Lesson5_2/Program.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Lesson5_2
+{
+    public class Car
+    {
+        public string name;
+        private CarPart[] parts;
+
+        public Car(string name, CarPart[] parts)
+        {
+            this.name = name;
+            this.parts = parts;
+        }
+
+        public int getPartsPrice ()
+        {
+            int price = 0;
+            for (int i = 0; i < this.parts.Length; i++)
+            {
+                price += this.parts[i].getPrice();
+            }
+
+            return price;
+        }
+    }
+
+    public class CarPart
+    {
+        private string name;
+        private int price;
```

Comment:
@MariaOsminkina You could replace it with Public int Price { get; }
In this case - set this value you could only from the constructor, but get it from anywhere, and method GetPrice could be removed;